mcux/drivers/imx/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ zephyr_library_compile_definitions_ifdef(
 )
 
 zephyr_library_sources_ifdef(CONFIG_COUNTER_MCUX_GPT	fsl_gpt.c)
+zephyr_library_sources_ifdef(CONFIG_COUNTER_MCUX_PIT    fsl_pit.c)
```

Comment:
Sorry for the nitpick, but please replace the spaces with a tab to match the rest of the file.

Reply:
nice catch, my default setting is 4 whitespaces, this is not good for C code